pkg/limitador/k8s_objects.go

```diff
@@ -92,6 +91,10 @@ func LimitadorDeployment(limitador *limitadorv1alpha1.Limitador) *appsv1.Deploym
 						{
 							Name:  "limitador",
 							Image: image,
+							Command: []string{
+								"limitador-server",
+								fmt.Sprintf("%s%s", LimitadorCMMountPath, LimitadorConfigFileName),
```

Comment:
Does this requires some escaping? Are there any special characters, like space (` `) that need escaping?

Reply:
No, sorry, I actually thought I had approved _while leaving the comment_, but looks like I didn't 🤦 
It was an actual question… not a "didn't you forget to escape?"